include/mbedtls/debug.h

```diff
@@ -75,6 +75,16 @@
 
 #endif /* MBEDTLS_DEBUG_C */
 
+#define MBEDTLS_SSL_CHK_BUF_PTR_WITH_DEBUG( cur, end, need )		\
+    do {								\
+        if( mbedtls_ssl_chk_buf( ( cur ), ( end ), ( need ) ) != 0 )	\
+        {								\
+            MBEDTLS_SSL_DEBUG_MSG( 1, ( "buffer too small: cur %p, end %p, want %u)", \
```

Comment:
Please note that giving the debug message a priority of 1 is very high, you will make the output very verbose. I suggest increasing this value to 3 or something like that.